client/src/entrypoints/admin/wagtailadmin.js

```diff
@@ -13,6 +13,14 @@ document.addEventListener('DOMContentLoaded', () => {
   initTabs();
   initSidePanel();
   initCollapsiblePanels();
+
+  const editHandlerElement = document.querySelector('[data-edit-handler]');
+  if (editHandlerElement) {
+    const editHandlerJson =
+      editHandlerElement.getAttribute('data-edit-handler');
+    const packedEditHandler = JSON.parse(editHandlerJson);
```

Comment:
With `json_script|"w-edit-handler-data"` this becomes

```suggestion
  const editHandlerElement = document.getElementById("w-edit-handler-data");
  if (editHandlerElement) {
    const packedEditHandler = JSON.parse(editHandlerElement.textContent);
```